.circleci/config.yml

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright (c) 2021, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+version: 2.1
+
+executors:
+  daml-docker-executor:
+    docker:
+      - image: cimg/openjdk:11.0
+
+commands:
+  install_sdk:
+    description: "Install Daml SDK"
+    steps:
+      - run:
+          name: Install Daml SDK
+          command: |
+            curl -sSL https://get.daml.com/ | sh /dev/stdin ${SDK_VERSION}
+            # idea from https://circleci.com/docs/2.0/env-vars/
+            >> $BASH_ENV echo 'export PATH="$HOME/.daml/bin:$PATH"'
+  install_yq:
+    description: "Install yq from binary"
+    steps:
+      - run:
+          name: Install yq
+          command: |
+            mkdir -p $HOME/yq
+            curl -L https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -o $HOME/yq/yq &&\
+              chmod +x $HOME/yq/yq &&\
+              echo 'export PATH=$HOME/yq/yq:$PATH' >> $BASH_ENV
+  install_gh:
+    description: "Install GitHub Cli"
+    steps:
+      - run:
+          name: Install gh
+          command: |
+            curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+            echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
```

Comment:
Shouldn't `| sudo tee filename > /dev/null` just be the same as `> filename`?

Reply:
See comment above - this comes directly from Github's recommended installation instructions. Your recommendation does create the identical file content as per the `config.yml` file but the file permissions are not the same (the `sudo tee` makes the file owned by `root`). 

As per above, I don't think its worth spending time investigating this and the effect of file ownership has on installing GitHub Cli.